lib/features/add_property/view_model/add_property_cubit.dart

```diff
@@ -16,12 +17,14 @@ class AddPropertyCubit extends Cubit<AddPropertyState> {
 
   PropertyTypes get propertyType => _type;
   set propertyType(PropertyTypes type) {
+    isAddPropertyFlowInteracted = true;
```

Comment:
Only enable this if the property type has changed from default type

```
if (_type != type) {
    // enable interaction here
}
```

Reply:
Fixed following suggestion